src/config.ts

```diff
@@ -145,6 +156,29 @@ function normalizeConfig(config: SnowpackConfig): SnowpackConfig {
     const isDetailedObject = config.webDependencies && typeof config.webDependencies === 'object';
     config.source = isDetailedObject ? 'pika' : 'local';
   }
+  normalizeEnv(config);
+  return config;
+}
+
+/**
+ * Prepares environment variable config for inlining purposes.
+ *
+ * @param config
+ */
+function normalizeEnv(config: SnowpackConfig) {
+  const reducer = (src: any) => (acc: any, id: string) => {
+    const index = id.indexOf('=');
+    const [key, val] = index > 0 ? [id.substr(0, index), id.substr(index + 1)] : [id, src[id]];
+    acc[`process.env.${key}`] = `"${val}"`;
+    return acc;
+  };
+  const env = config.installOptions.env;
+  const result: any = {
```

Comment:
can you give this a key signature? `result: {[env: string]: string} = ...`

Reply:
I've added a new type alias `EnvVarReplacements`...